conf.pri.msvc

```diff
@@ -0,0 +1,40 @@
+# Adapt these paths
+# Point these to the include folders
+INCLUDEPATH += $$quote(C:/qBittorrent/boost_1_64_0)
+INCLUDEPATH += $$quote(C:/qBittorrent/RC_1_1/include)
+INCLUDEPATH += $$quote(C:/qBittorrent/Zlib/include)
+INCLUDEPATH += $$quote(C:/qBittorrent/openssl/include)
+# Point these to the lib folders
+LIBS += $$quote(-LC:/qBittorrent/boost_1_64_0/stage/lib)
+LIBS += $$quote(-LC:/qBittorrent/RC_1_1/bin/path-according-to-the-build-options-chosen)
+LIBS += $$quote(-LC:/qBittorrent/zlib/lib)
+LIBS += $$quote(-LC:/qBittorrent/openssl/lib)
+
+# Adapt the lib names/versions accordingly
+CONFIG(debug, debug|release) {
+    LIBS += libtorrentd.lib \
+            libboost_system-vc140-mt-sgd-1_64.lib
+}
+else {
+    LIBS += libtorrent.lib \
+            libboost_system-vc140-mt-s-1_64.lib
+}
+
+LIBS += advapi32.lib shell32.lib crypt32.lib User32.lib ole32.lib
```

Comment:
This are mandatory system libraries, can be moved to winconf.pri, with mingw system libraries too and use condition, like:
```js
win32-g++* {
    LIBS += libadvapi32 libshell32 libuser32 libole32 libpowrprof
    LIBS += libcrypto libssl libwsock32 libws2_32 libz libiconv
}
else {
    LIBS += advapi32.lib shell32.lib crypt32.lib User32.lib ole32.lib PowrProf.lib
}
```

Or if I think about that, they can be moved all to winconf.pri, here can be problem, that openssl 1.1 has other names for library files.
Or libeay32.lib and ssleay32.lib can stay in example conf.pri.msvc

Reply:
>libeay32.lib and ssleay32.lib can stay in example

I think Yes.